src/NuGet.Core/NuGet.ProjectModel/TargetFrameworkInformation.cs

```diff
@@ -17,14 +17,23 @@ public class TargetFrameworkInformation : IEquatable<TargetFrameworkInformation>
 
         /// <summary>
         /// A fallback PCL framework to use when no compatible items
-        /// were found for <see cref="FrameworkName"/>.
+        /// were found for <see cref="FrameworkName"/>. This check is done
+        /// per asset type.
         /// </summary>
+        /// <remarks>Deprecated. Use <see cref="AssetTargetFallback" /> instead.</remarks>
+        /// <see cref="Imports" /> cannot be used with <see cref="AssetTargetFallback" />.
         public IList<NuGetFramework> Imports { get; set; } = new List<NuGetFramework>();
```

Comment:
Discussed offline, the property is deprecated for projects but internally we still need to support reading this property so that we can error appropriately when it is used. I'm going to leave the attribute off to avoid build warnings.